packages/@webex/plugin-meetings/test/unit/spec/roap/request.ts

```diff
@@ -1,114 +0,0 @@
-import {assert} from '@webex/test-helper-chai';
```

Comment:
why has this test file been renamed from ts to js?

Reply:
i think when this was pointing to master and didn't have the beta changes in, it was js and when i pulled the changes from  beta and point it to beta, i ended up with 2 files, i didn't notice i deleted the ts file.

changed back to ts